### DISCUSSION TITLE
schedule speed test

### DISCUSSION THREAD

**Discussion by sadden3194** [alexjustesen/speedtest-tracker#662] [Q&A] [answered]
Hi all!

I would like to have 4 test per day:

```
0:00 AM
6:00 AM
12:00 AM
18:00 AM
```

I configure the Speedtest settings "Speedtest schedule like this:
`0 */6 * * *`

Is that correct?

**Comment by alexdelprete** [comment]
[crontab guru](https://crontab.guru/#0_*/6_*_*_*) says it's correct:

![image](https://github.com/alexjustesen/speedtest-tracker/assets/7027842/c4b42e55-fb30-4d47-afe0-f960b8042a75)

**Reply by sadden3194** [reply]
yes it says every 6 hours.
but not from which time as startpoint.
in my situation: 0:00 AM

**Reply by alexdelprete** [reply, marked as answer]
it says "At minute 0 past every 6th hour". More clear than this? Impossible. :)

it also says "next start at 18:00:00".

You should read things more carefully...
